liquid-rust-driver/src/lower/local.rs

```diff
@@ -9,7 +8,7 @@ impl<'tcx> Lower<'tcx> for mir::Local {
     type Output = Local;
 
     fn lower(&self, _lcx: LowerCtx<'tcx>) -> LowerResult<Self::Output> {
-        Ok(Local::new(self.index()))
+        Ok(*self)
```

Comment:
Same here, I think we can just drop the `impl`.

Reply:
yes